doc/htmldoc/synapses/connection_management.rst

```diff
@@ -588,20 +588,20 @@ columns and :math:`m` rows.
 
     # To only connect pairs with a nonzero weight, we use array indexing
     # to extract the weights and post-synaptic neurons.
-          nonzero_indices = numpy.where(weights != 0)[0]
-          weights = weights[nonzero_indices]
-          post = B[nonzero_indices]
+        nonzero_indices = np.where(weights != 0)[0]
```

Comment:
I'd argue that these should remain ad numpy, for the reason that it's explicit. We don't set up `import numpy as np` anywhere in the code examples on this page. So I'd suggest changing the first `np.array` to `numpy.array`

Reply:
I agree with your thought process, thanks for the callout.
There was another line prior to this line that was using `np` and I've amended that to `numpy` as well.